Makefile

```diff
@@ -66,13 +66,13 @@ go-deps:
 	@cp $(codegeneratorRoot)/generate-internal-groups.sh $(codegeneratorTarget) && chmod +x $(codegeneratorTarget)/generate-internal-groups.sh
 
 install-tools:
-	GO111MODULE=on go build -o $(GOPATH)/bin/golangci-lint -mod=vendor ./vendor/github.com/golangci/golangci-lint/cmd/golangci-lint
-	GO111MODULE=on go build -o $(GOPATH)/bin/gosec -mod=vendor ./vendor/github.com/securego/gosec/cmd/gosec
+	GO111MODULE=on go build -o $(GOPATH)/bin/golangci-lint ./vendor/github.com/golangci/golangci-lint/cmd/golangci-lint
+	GO111MODULE=on go build -o $(GOPATH)/bin/gosec ./vendor/github.com/securego/gosec/cmd/gosec
 
 # Run verification steps
 # Example:
 #    make verify
-verify: install-tools
+verify: go-deps install-tools
```

Comment:
why this here?

Reply:
there's no need here, it's static code analysis on our code, not vendored one

Reply:
```
hack/verify-codegen.sh
build k8s.io/code-generator/cmd/defaulter-gen: cannot load github.com/spf13/pflag: open /go/src/github.com/openshift/machine-config-operator/vendor/k8s.io/code-generator/vendor/github.com/spf13/pflag: no such file or directory
make: *** [verify] Error 1 
```
and with it:
```
# go mod does not vendor in scripts so we need to get them manually...
cp: '/go/src/github.com/openshift/machine-config-operator/vendor/k8s.io/code-generator/generate-groups.sh' and './vendor/k8s.io/code-generator/generate-groups.sh' are the same file
make: *** [go-deps] Error 1
--- 
```